tests/queries/0_stateless/00800_function_java_hash.reference

```diff
@@ -1,3 +1,13 @@
+123
+-123
+123
+-123
+123
+-123
+123
+122
+-539222985
+-539222986
```

Comment:
Is there any online playground or a quick code snippet to validate that this is definitely the same result as Java gives?

Reply:
Unfortunately I did not find any online tool.  But we can generate Java hash code simplly by` Long.hashCode()`.